MobileApplication/MobileApplication/ViewModels/AddFileAndImage.cs

```diff
@@ -0,0 +1,45 @@
+﻿using MobileApplication.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using Xamarin.Essentials;
+
+namespace MobileApplication.ViewModels
+{
+    public  class AddFileAndImage
```

Comment:
Maybe the better name could be something like: `MediaUploader` or `TransactionUploader` with methods something like UploadReceipt( Image ) and UploadBankStatment( csvFile )

Reply:
I will change it.